.github/workflows/latest_tags.yml

```diff
@@ -0,0 +1,116 @@
+      # orgs we will need to restructure this workflow
+      owner: celestiaorg
+    # Define the outputs for other steps to access
+    outputs:
+      celestia-app_latest_tag: ${{ steps.set_outputs.outputs.celestia-app_latest_tag }}
+      celestia-app_latest_sha: ${{ steps.set_outputs.outputs.celestia-app_latest_sha }}
+      celestia-core_latest_tag: ${{ steps.set_outputs.outputs.celestia-core_latest_tag }}
+      celestia-core_latest_sha: ${{ steps.set_outputs.outputs.celestia-core_latest_sha }}
+      celestia-node_latest_tag: ${{ steps.set_outputs.outputs.celestia-node_latest_tag }}
+      celestia-node_latest_sha: ${{ steps.set_outputs.outputs.celestia-node_latest_sha }}
+    steps:
+      - name: Log target owner/repo
+        run: "echo Pulling info from: '${{ env.owner }}/${{ matrix.repo }}'"
+
+      - name: Query /releases/latest endpoint
+        uses: octokit/request-action@v2.x
+        id: get_latest_release
+        with:
+          route: GET /repos/{owner}/{repo}/releases/latest
+          owner: ${{ env.owner }}
+          repo: ${{ matrix.repo }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Log latest tag
+        run: "echo latest tag: '${{ fromJSON(steps.get_latest_release.outputs.data).tag_name }}'"
+
+      - name: Query /git/ref/tag endpoint
+        uses: octokit/request-action@v2.x
+        id: get_latest_commit
+        with:
+          route: GET /repos/{owner}/{repo}/git/ref/tags/${{ fromJSON(steps.get_latest_release.outputs.data).tag_name }}
+          owner: ${{ env.owner }}
+          repo: ${{ matrix.repo }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Log latest sha
+        run: "echo latest commit: '${{ fromJSON(steps.get_latest_commit.outputs.data).object.sha }}'"
+
+      - name: Set outputs
+        id: set_outputs
+        run: |
+          echo "${{matrix.repo}}_latest_tag=${{ fromJSON(steps.get_latest_release.outputs.data).tag_name }}" >> $GITHUB_OUTPUT
+          echo "${{matrix.repo}}_latest_sha=${{ fromJSON(steps.get_latest_commit.outputs.data).object.sha }}" >> $GITHUB_OUTPUT
+
+  # This job creates a PR if there were changes. The changes are created when
+  # the workflow writes to the version files. This means that the workflow
+  # should be able to be run and will just be a no-op if the is no new release.
+  # Either way, since a PR is created, this projects breaking the docs on main.
+  createPR:
+    needs: logLatestRelease
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v3
+      - name: Write latest tags and commit SHAs to txt files
+        # This is the code that writes to a file. If we need to update the file
+        # layout, contents, or file name, do it here.
+        run: |
+          CONTENTS=$(cat << EOF
+          const ${{ inputs.network }}Versions = Object.freeze({
+            "golang": "1.19.1",
```

Comment:
[follow up]
You might want to pull the golang version out into a separate file since the golang version is more by repo not network.  Plus you don't want this hardcoded value overwritting any future golang version changes.

Reply:
looks like you already have the versions file and use it for golang so this line could be removed
```suggestion
```